DDR4 OC Guide.md

```diff
@@ -438,26 +438,44 @@ Some terminology:
    * Make sure your UEFI/BIOS is up to date.
 
-2. On Intel, start with 1.15 V VCCSA and VCCIO.  
+2. On Intel, set command rate (CR) to 2T if it isn't already and set tCCDL to 8 (Asus UEFIs don't expose this timing).
+   On AMD, set Gear Down Mode to Enabled if it isn't already.
+
+3. On Intel, start with 1.15 V VCCSA and VCCIO. For ADL, VCCIO does not exist. Note that if you have an Alder Lake non-K SKU, VCCSA will be locked and your overclock potential will be limited.
    On AMD, start with 1.10 V SOC, 0.95 V VDDP, 0.95 V VDDG CCD, and 1.05 V VDDG IOD.
+   * These voltgages will need to be increased as you raise frequency or tighten timings. Refer to [Integrated Memory Controller (IMC)](#integrated-memory-controller-imc) for recommended voltages.
    * SOC voltage might be named differently depending on the manufacturer.
      * Asrock: CPU VDDCR_SOC Voltage. If you can't find that, you can use SOC Overclock VID hidden in the AMD CBS menu.
        * [VID values](https://www.reddit.com/r/Amd/comments/842ehb/asrock_ab350_pro4_guide_bios_overclocking_raven/).
      * Asus: VDDCR SOC.
      * Gigabyte: (Dynamic<sup>1</sup>) Vcore SOC.
        * <sup>1</sup>Dynamic Vcore SOC is found on certain Gigabyte motherboards and is an offset voltage. Therefore, the base voltage can change automatically when increasing DRAM frequency. For example, +0.100 V at DDR4-3000 might result in 1.10 V actual, but +0.100V at DDR4-3400 might result in 1.20v actual.
      * MSI: CPU NB/SOC.
-3. Set DRAM voltage to 1.40 V. If you're using ICs that roll over above 1.35 V, set 1.35 V.
+4. To find what voltage to use for your IC, refer to the [maximum recommended daily voltage section](#maximum-recommended-daily-voltage). If you're using ICs that roll over above 1.35 V, set 1.35 V.
    * "Roll over" means that the IC becomes more unstable as you increase the voltage, sometimes to the point of not even POSTing.
    * ICs that are known to roll over above 1.35 V include but are not limited to: 8 Gb Samsung C-die and older Micron/SpecTek ICs (before M8E).
-   * To find what voltage to use for your IC, refer to the [maximum recommended daily voltage section](#maximum-recommended-daily-voltage).
 
-4. Set primary timings to 16-20-20-40 (tCL-tRCD-tRP-tRAS) and tCWL to 16.
-   * Most ICs need loose tRCD and/or tRP, so I recommend 20.
+5. Set loose primary timings. See the table below.
+
+    |Frequency|tCL|tRCD|tRP|tRAS|tCWL|
+    |---|---|---|---|---|---|
+    |<=3200|16|20|20|40|16|
+    |3201-3600|18|22|22|44|18|
+    |3601-4000|20|24|24|48|20|
+    |4001-4400|22|26|26|52|22|
+    |4400+|24|28|28|56|24|
+
+   * Some ICs may not boot with very loose primary timings to begin with so it is recommended to loosen timings before the frequency range is met while increasing the frequency in the next step as detailed in the table above. 
```

Comment:
This sentence is a bit confusing

What do you mean by "before the frequency range is met"?

I'd suggest splitting this sentence into 2 to make it easier to read